### DISCUSSION TITLE
clear/reset authentication URL

### DISCUSSION THREAD

**Discussion by kjustie** [homebridge-simplisafe3/homebridge-simplisafe3#291] [Q&A] [answered]
How can I clear the SimpliSafe authentication URL, in order to enter a new one?  

I'm getting "SimpliSafe login failed with error: Error: Failed refreshing token: Error: Request failed with status code 403," but the plug-in settings say authentication has already been set up (which is correct), and while I can "Launch SimpliSafe Login" to get a new URL, the field to enter it in the settings is not visible -- since it already has (an apparently non-working) one.

thanks!

**Comment by shamoon** [comment, marked as answer]
Hey @kjustie here's a video that shows how it should work, if you've already authenticated theres a link to show the instructions and then clicking the button should reveal that URL field. Let me know if this clarifies.

https://user-images.githubusercontent.com/4887959/152275267-80dc0174-3085-4867-819b-4e8e5b7bb6db.mov

**Reply by kjustie** [reply]
It was because the URL field DIDN'T appear when I clicked "Launch SimpliSafe Login" -- on multiple attempts -- that I wrote in the first place. Having been through the process before -- when it first became necessary, and then when moving to a new Homebridge server a couple months ago --  I was very familiar with how it should work.  But last night, it correctly opened a new browser window to the SimpleSave authentication page, but when I'd come back after getting the URL, the URL field was definitely not there - in multiple tries.   But when I tried it again after getting your reply (and again this morning, just to see what would happen), it did work as it should have, so mission accomplished.    So, not sure what could have been different that prevented it from working correctly initially, but thanks for your help!

**Reply by shamoon** [reply]
Yea thats definitely odd. If you notice that again it might be helpful to look in the browser javascript console for errors and / or homebridge logs.
